ticker/src/test/java/com/robinhood/ticker/TickerCharacterListTest.java

```diff
@@ -17,23 +17,39 @@ public void test_initialization() {
     @Test
     public void test_getCharacterIndices() {
         final TickerCharacterList list = new TickerCharacterList("012");
-        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('0', '1');
+        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('0', '1', TickerView.DIRECTION_ANY);
         assertEquals(1, indices.startIndex);
         assertEquals(2, indices.endIndex);
     }
 
     @Test
     public void test_getCharacterIndicesWraparound() {
         final TickerCharacterList list = new TickerCharacterList("012");
-        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('2', '0');
+        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('2', '0', TickerView.DIRECTION_ANY);
         assertEquals(3, indices.startIndex);
         assertEquals(4, indices.endIndex);
     }
 
     @Test
     public void test_getCharacterIndicesWraparound2() {
         final TickerCharacterList list = new TickerCharacterList("012");
-        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('0', '2');
+        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('0', '2', TickerView.DIRECTION_ANY);
+        assertEquals(4, indices.startIndex);
+        assertEquals(3, indices.endIndex);
+    }
+
+    @Test
+    public void test_getCharacterIndicesForcedDown() {
+        final TickerCharacterList list = new TickerCharacterList("012");
+        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('2', '0', TickerView.DIRECTION_DOWN);
+        assertEquals(3, indices.startIndex);
+        assertEquals(4, indices.endIndex);
+    }
+
+    @Test
+    public void test_getCharacterIndicesForcedUp() {
+        final TickerCharacterList list = new TickerCharacterList("012");
+        final TickerCharacterList.CharacterIndices indices = list.getCharacterIndices('0', '2', TickerView.DIRECTION_UP);
```

Comment:
forced down and forced up tests should be the same, so here we should try to go from `2` to `0`, and the start and end indices would be 3 and 1 respectively.

Reply:
Good call. I'll add similar `test___2()` duplicate functions